server/src/index_db/generate_version.rs

```diff
@@ -13,12 +13,13 @@ impl From<PostgresError> for Error {
     }
 }
 
-pub fn generate_version(client: &mut PostgresClient) -> Result<i64, Error> {
```

Comment:
This logic should be part of a database transaction. Potential race condition: read time for edit, read time for updates, get updates, write edit at past time (never gets sent in update poll)